nrf5340-net-hal/README.md

```diff
@@ -0,0 +1,33 @@
+# Hardware Abstration Layer for the Nordic nRF5340
+
+This crate is a Hardware Abstraction Layer (HAL) for the Nordic nRF5340. It
+wraps the PAC (`nrf5340-pac`) and provides high level wrappers for the chip's
+peripherals.
```

Comment:
The readme refers to a nonexistent crate. Since most HAL crates don't have an extra readme, the file can probably just be deleted.